Directory.Packages.props

```diff
@@ -4,19 +4,22 @@
   <PropertyGroup>
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
     <CentralPackageTransitivePinningEnabled>true</CentralPackageTransitivePinningEnabled>
-    <XunitV3LibraryVersion>1.0.0</XunitV3LibraryVersion>
+    <XunitV3LibraryVersion>2.0.0</XunitV3LibraryVersion>
   </PropertyGroup>
   <ItemGroup>
     <PackageVersion Include="Microsoft.CodeAnalysis.PublicApiAnalyzers" Version="3.3.4" />
-    <PackageVersion Include="xunit.v3.extensibility.core" Version="1.0.1" />
+    <PackageVersion Include="xunit.v3.extensibility.core" Version="2.0.0" />
   </ItemGroup>
   <ItemGroup Condition="'$(MSBuildProjectName)'=='Xunit.StaFact'">
     <PackageVersion Update="xunit.v3.extensibility.core" Version="$(XunitV3LibraryVersion)" />
   </ItemGroup>
   <ItemGroup Label="Library.Template">
     <PackageVersion Include="Microsoft.NET.Test.Sdk" Version="17.12.0" />
+    <PackageVersion Include="xunit.analyzers" Version="1.20.0" />
     <PackageVersion Include="xunit.runner.visualstudio" Version="3.0.1" />
-    <PackageVersion Include="xunit.v3" Version="1.0.1" />
+    <PackageVersion Include="xunit.v3" Version="2.0.0" />
+    <PackageVersion Include="xunit.v3.assert" Version="2.0.0" />
+    <PackageVersion Include="xunit.v3.runner.inproc.console" Version="2.0.0" />
```

Comment:
The _new_ ones shouldn't be placed in the `Library.Template` labeled ItemGroup. They should be placed in the non-conditioned one above.